rollup.config.js

```diff
@@ -0,0 +1,15 @@
+import { terser } from "rollup-plugin-terser";
```

Comment:
I'd argue that you don't need the `graph-es-sdk.js` file as the other bundled js files work just as fine in the browsers. A single file with all modules defeats the purpose of modules.